.github/workflows/build.yml

```diff
@@ -0,0 +1,611 @@
+# build PythonEmbedInR, and deploy to S3 RAN server if conditions merit.
+# at this point experimental
+
+name: build
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+      - 'develop'
```

Comment:
Note:  We normally build when we merge a developer's feature branch into the 'develop' branch of the Sage Bio repo'.